packages/redux-routine/src/test/is-iterator.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import isIterator from '../is-iterator';
+
+describe( 'isIterator', () => {
+	test.each( [
+		[ undefined ],
+		[ null ],
+		[ 10 ],
+		[ 'foo' ],
+		[ [ 0, 1, 2, 3 ] ],
```

Comment:
These checks will both fail because these _are_ iterators!

Reply:
@aduth Looking at this module, I'd expect `redux-routine` to handle most of these types without issue as long as they contain actions.

Notable exceptions would be `string` and `TypedArray`, which couldn't contain action objects to the best of my knowledge.

From the example in the README:

```js
function* retrieveTemperature() {
	const result = yield { type: 'FETCH_JSON', url: 'https://' };
	return { type: 'SET_TEMPERATURE', temperature: result.value };
}
store.dispatch( retrieveTemperature() );

// It seems like we could also:

function severalActionsInAnIterable() {
	return [
		{ type: 'FIRST_ACTION' },
		{ type: 'SECOND_ACTION' },
	];
}
store.dispatch( severalActionsInAnIterable() );
```

I'm curious to hear your thoughts on this.